resources/startup.sh

```diff
@@ -34,33 +46,60 @@ TEMPORARY_ADMIN_GROUP=$(doguctl random)
 TEMPORARY_ADMIN_USER=$(doguctl random)
 TEMPORARY_ADMIN_PASSWORD=$(doguctl random)
 
-function import_quality_profiles_if_present {
-  AUTH_USER=$1
-  AUTH_PASSWORD=$2
+function areQualityProfilesPresent() {
```

Comment:
This function name does only apply to the quality profile directory and not to SonarQube in general.
```suggestion
function areQualityProfilesPresentForImport() {
```

Reply:
point taken